index.html

```diff
@@ -92,56 +93,77 @@ <h4 class="card-heading">Multi-Post Stories Gain+Glory</h4>
-            <li><button type="button">Ruby on Rails</button></li>
-            <li><button type="button">RSpec</button></li>
-            <li><button type="button">CapyBara</button></li>
-            <li><button type="button">Selenium</button></li>
-        </ul>
-    </div>   
-</li>
-<li class="card2">
-    <img src="img/icon-skills.svg" alt="icon">
-<div class="languages">
-    <h5 class="card-heading heading-color">Skills</h5>
-    <ul class="card-buttons background-btn">
-        <li><button type="button">CodeKit</button></li>
-        <li><button type="button">GitHub</button></li>
-        <li><button type="button">CodePen</button></li>
-        <li><button type="button">GitLab</button></li>
-        <li><button type="button">Terminal</button></li>
-    </ul>
-</div>   
-</li>
+            <img src="img/icon-languages.svg" alt="icon">
+                <div class="languages">
+                    <h5 class="card-heading heading-color">Languages</h5>
+                    <ul class="card-buttons background-btn">
+                        <li><button type="button">Javascript</button></li>
+                        <li><button type="button">Ruby</button></li>
+                        <li><button type="button">HTML</button></li>
+                        <li><button type="button">CSS</button></li>
+                    </ul>
+                </div>
+        </li>
+        <li class="card2">
+            <img src="img/icon-frameworks.svg" alt="icon">
+                <div class="languages">
+                    <h5 class="card-heading heading-color">Frameworks</h5>
+                    <ul class="card-buttons background-btn">
+                        <li><button type="button">Bootstrap</button></li>
+                        <li><button type="button">Ruby on Rails</button></li>
+                        <li><button type="button">RSpec</button></li>
+                        <li><button type="button">CapyBara</button></li>
+                        <li><button type="button">Selenium</button></li>
+                    </ul>
+                </div>   
+        </li>
+        <li class="card2">
+            <img src="img/icon-skills.svg" alt="icon">
+                <div class="languages">
+                    <h5 class="card-heading heading-color">Skills</h5>
+                    <ul class="card-buttons background-btn">
+                        <li><button type="button">CodeKit</button></li>
+                        <li><button type="button">GitHub</button></li>
+                        <li><button type="button">CodePen</button></li>
+                        <li><button type="button">GitLab</button></li>
+                        <li><button type="button">Terminal</button></li>
+                    </ul>
+                </div>   
+        </li>
     </ul>
     </section>
 
 </main>
+
+<footer class="footing">
+<h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```

Comment:
stick to the design templates in Figma as much as possible, this includes the font, colours, images and spacing (margin/padding). use the right font family for this text.

Reply:
Thank you for that as well it has been handled

Reply:
This issue was successfully addressed. :+1: